src/main/java/io/jenkins/plugins/opentelemetry/JenkinsOpenTelemetryPluginConfiguration.java

```diff
@@ -71,6 +75,8 @@
 
     private transient OpenTelemetrySdkProvider openTelemetrySdkProvider;
 
+    private boolean dontExportOtelConfigurationAsEnvironmentVariables;
```

Comment:
a default value instead? Like `false`

Reply:
I wanted to enable this by default as I feel it's not a security problem in most cases because it's just telemetry.

`false` is the default value as the config attribute is "Don't export Otel Configuration as Environment Variables". Due to the way Jenkins XStream works, there would be some work to make it `true` by default, I would define the opposite variable instead: `exportOtelConfigurationAsEnvironmentVariables`.